src/Workleap.DomainEventPropagation.Publishing/EventPropagationPublisherOptions.cs

```diff
@@ -12,10 +12,20 @@ public sealed class EventPropagationPublisherOptions
 {
     internal const string SectionName = "EventPropagation:Publisher";
 
-    // EventPropagationClient was the legacy name when only one kind of topic was supported
-    internal const string CustomTopicClientName = "EventPropagationClient";
-
-    internal const string NamespaceTopicClientName = "NamespaceTopicClient";
+    /// <summary>
+    /// This is the name of the underlying named options for <see cref="Azure.Messaging.EventGrid.EventGridPublisherClientOptions"/> and <see cref="Azure.Messaging.EventGrid.Namespaces.EventGridClientOptions"/>.
+    /// They are used when creating the corresponding <see cref="Azure.Messaging.EventGrid.EventGridPublisherClient"/> and <see cref="Azure.Messaging.EventGrid.Namespaces.EventGridClient"/>.
+    /// </summary>
+    /// <example>
+    /// Use this option name to customize the Event Grid clients.
+    /// <code>
+    /// services.Configure&lt;EventGridPublisherClientOptions&gt;(EventPropagationPublisherOptions.EventGridClientName, options =>
+    /// {
+    ///     // ...
+    /// }).
+    /// </code>
+    /// </example>
+    public const string EventGridClientName = "DomainEventPropagation";
```

Comment:
This is how it's displayed in an IDE:

![image](https://github.com/gsoft-inc/wl-domain-event-propagation/assets/14242083/19956196-1379-46e2-b65f-f0cd12c4d3d0)